src/cpp/subscriber/StatisticsParticipantListener.cpp

```diff
@@ -595,12 +595,6 @@ void StatisticsParticipantListener::on_publisher_discovery(
 {
     static_cast<void>(participant);
 
-    // Filter out other statistics writers
-    if (is_statistics_builtin(info.info.guid().entityId))
```

Comment:
Be careful erasing this. Check in `StatisticsParticipantListener::on_subscriber_discovery` that it is checked for removing the monitor subscriptions. Probably we need that here as well.

Reply:
We decided to deactivate the publication of the backend's statistics from within the code, since it was causing issues and it is not expected to be of any use. Hence there is no longer need to filter our own statistics writers.

Reply:
Add a comment or warning that explains why own statistics writers does not need to be filtered.